core/templates/dev/head/pages/state_editor/StateInteractionEditorDirective.js

```diff
@@ -213,8 +215,16 @@ oppia.directive('stateInteractionEditor', [
                       UrlInterpolationService.getInteractionThumbnailImageUrl);
 
                     $scope.INTERACTION_SPECS = INTERACTION_SPECS;
-                    $scope.ALLOWED_INTERACTION_CATEGORIES = (
-                      GLOBALS.ALLOWED_INTERACTION_CATEGORIES);
+
+                    // Checks that the modal is invoked by exploration_editor or
+                    // skill_editor, topic_editor. 
+                    if ($window.location.href.indexOf('/questions') == -1) {
+                      $scope.ALLOWED_INTERACTION_CATEGORIES = (
+                        ALLOWED_INTERACTION_CATEGORIES);
+                    } else {
```

Comment:
In case current path does not match any of the given paths, this should give an error.

Reply:
You can use them directly like `skill_editor`, `create` etc.

Reply:
Done. PTAL.